relay-general/src/store/transactions/rules.rs

```diff
@@ -149,16 +167,21 @@ impl TransactionNameRule {
         }
     }
 
-    /// Returns `true` if the current rule pattern matches transaction, expected transaction
-    /// source, and not expired yet.
-    pub fn matches(&self, transaction: &str, info: &TransactionInfo) -> bool {
-        let now = Utc::now();
+    /// Returns `true` if the transaction info has a matching transaction source and
+    /// the transaction name matches the rule (see
+    /// [`TransactionNameRule::matches`]).
```

Comment:
I belive this link doesn't really work, but I spent too much time already and didn't manage to point to that method properly. Any ideas?

Reply:
Does `Self` work?